### ISSUE TITLE
Update to v0.3.0 failed, services start but are not reachable, docker-compose problem?

### ISSUE THREAD

**Issue by esackbauer** [getumbrel/umbrel#379]
I tried the update over the GUI, from v0.2.15. The progress bar was stuck, and after like 30 minutes I had a look, I have a fast machine and starting/stopping Umbrel usually takes like 30 seconds.

Content of update-status.json:

> {"state": "failed", "progress": 100, "description": "An unexpected error occured", "updateTo": ""}

The logs indicate that everything "0?-run.sh" went through without throwing errors:

> Successfully installed Umbrel v0.3.0
> == End Update Script 03-run.sh ==
> 
> Deleting cloned repository
> Removing lock

When inspecting the docker-compose.yml something seems wrong, no $variables like IP addresses were replaced with actual IP addresses or ports.
Could this be the culprit? How to recreate a proper docker-compose file?

Debian 10.7 amd64
Docker 20.10.1 (via debian repository)
docker-compose 1.27.4

**Comment by robertclarkson** [comment]
Also having issues with updating. The updating icon is sat spinning and my update-status json looks like this
```
{
  description: "Successfully installed Umbrel v0.2.15"
  progress: 100
  state: "success"
  updateTo: "v0.3.0"
}
```

**Comment by Wox1** [comment]
Same Problem here. Update to 0.3.0 - starting over the web-login. Then the update is starting and about ~ 5Minut all stops. And the Umbrel is not reachable...

**Comment by paulo03** [comment]
Same here. Attempted 0.3.0 over GUI, now I cannot access the GUI at umbrel.local, but update-status.json says: `{"state": "success", "progress": 100, "description": "Successfully installed Umbrel v0.3.0", "updateTo": ""}`

**Comment by robertclarkson** [comment]
Is it ok to download a new image of umbrel and overwrite your existing image on the SD card?

**Comment by esackbauer** [comment]
@Wox1 @paulo03 can you please check your docker-compose.yml file if there are actual IP addresses and ports in it?

**Comment by AaronDewes** [comment]
> Is it ok to download a new image of umbrel and overwrite your existing image on the SD card?

Yes, that works

**Comment by paulo03** [comment]
@esackbauer My issue has been resolved. The solution was to run this (got it from @Umbrel over twitter):

`sudo ~/umbrel/scripts/update/update --repo lukechilds/umbrel#fix-ip-collisions`

**Comment by mayankchhabra** [comment]
Sorry about the failed update, guys. There's a networking issue in the latest release regarding IP collisions. 

Here’s an intermediary fix to get it working:

Step 1. SSH into your Umbrel (on Umbrel OS):

```
ssh umbrel@umbrel.local
```

Password: `moneyprintergobrrr`

Step 2. Manually OTA update to the repo with intermediary fix:

**If your Umbrel's local IP address is 10.0.x.x, run:**
```
sudo ~/umbrel/scripts/update/update --repo lukechilds/umbrel#fix-ip-collisions
```

**Otherwise, run:**
```
sudo ~/umbrel/scripts/update/update --ota
```

**Comment by robertclarkson** [comment]
Tried the first command, seemed to work, but after rebooting the node is unreachable on the web interface.

Trying the second command - it didnt work
```
umbrel@umbrel:~ $ sudo ~/umbrel/scripts/update/update --ota

=======================================
=============== UPDATE ================
=======================================
========== Stage: Download ============
=======================================

Creating lock
Cleaning up any previous mess
Downloading Umbrel 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     52      0 --:--:-- --:--:-- --:--:--    52

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
Running update install scripts of the new release
/home/umbrel/umbrel/scripts/update/update: line 109: cd: scripts/update: No such file or directory
ls: cannot access '*-run.sh': No such file or directory
Deleting cloned repository
Removing lock
```

**Comment by louneskmt** [comment]
@robertclarkson Run this instead of the second command:

echo '{"state": "installing", "progress": 10, "description": "Downloading Umbrel", "updateTo": "v0.3.0"}' > ~/umbrel/statuses/update-status.json && sudo ~/umbrel/scripts/update/update --ota

**Comment by esackbauer** [comment]
> sudo ~/umbrel/scripts/update/update --repo lukechilds/umbrel#fix-ip-collisions

That worked for me, thanks!!

**Comment by robertclarkson** [comment]
> @robertclarkson Run this instead of the second command:
> 
> ```
> echo '{"state": "installing", "progress": 10, "description": "Downloading Umbrel", "updateTo": "v0.3.0"}' > ~/umbrel/statuses/update-status.json && sudo ~/umbrel/scripts/update/update --ota
> ```

Thanks for the help, same error as before.

**Comment by mayankchhabra** [comment]
@robertclarkson you said the first command worked for you, right? 

Can you please try that again? If it doesn't work after a reboot, can you please run the debug script and paste the links to the logs it generates?

```
~/umbrel/scripts/debug --upload
```

**Comment by esackbauer** [comment]
Original issue solved by 0.3.1 update.

**Comment by robertclarkson** [comment]
@mayankchhabra  I think my issues are actually steming from a problem with my external SSD drive, which might be caused by an underpowered power supply. I havent had time to resolve yet and will be re-imaging from scratch. Thanks for the help!
